paimon-common/src/main/java/org/apache/paimon/fileindex/bitmap/BitmapFileIndex.java

```diff
@@ -49,39 +49,51 @@
 public class BitmapFileIndex implements FileIndexer {
 
     public static final int VERSION_1 = 1;
+    public static final int VERSION_2 = 2;
+
+    public static final String VERSION = "version";
+    public static final String INDEX_BLOCK_SIZE = "index-block-size";
+    public static final String ENABLE_BUFFERED_INPUT = "enable-buffered-input";
+    public static final String ENABLE_NEXT_OFFSET_TO_SIZE = "enable-next-offset-to-size";
```

Comment:
What is enable next offset to size? Under what circumstances can it not be size?